src/test/java/posters/cucumber/support/OpenPageFlows.java

```diff
@@ -6,60 +6,49 @@
 import com.xceptance.neodymium.util.Neodymium;
 
 import io.cucumber.java.en.Given;
-import io.qameta.allure.Step;
 import posters.pageobjects.pages.browsing.HomePage;
-import posters.pageobjects.pages.browsing.ProductdetailPage;
+import posters.pageobjects.pages.browsing.ProductDetailPage;
 import posters.pageobjects.pages.user.LoginPage;
 import posters.pageobjects.pages.user.RegisterPage;
 
 public class OpenPageFlows
 {
     @Given("^homepage is loaded$")
-    @Step("open home page")
-    public static HomePage homepage()
+    public static HomePage homePage()
     {
         // clear cookies to ensure a new session
         clearBrowserCookies();
         // open home page
         open(Neodymium.configuration().url());
-        HomePage homePage = new HomePage();
-        homePage.isExpectedPage();
-        return homePage;
+        return new HomePage().isExpectedPage();
     };
```

Comment:
@oomelianchuk Please remove the `;` after functions. They are not needed.

Reply:
Done